README.md

```diff
@@ -49,10 +49,147 @@ $ yarn lint
 # Run tests
 $ yarn test
 
-# Record new trafic with live API calls
+# Record new trafic with live API calls, more information bellow
 $ yarn test:record grid/path/to/test.ts
 ```
 
+Jest in station is configured to run tests inside `/grid` folder by default, with reporter located in `/jest/reporter` and can use groups to filter which tests to run (mainly used for tagging tests ready to continuous testing).
+
+```javascript
+module.exports = {
+  preset: 'ts-jest',
+  testEnvironment: 'node',
+  rootDir: './grid',
+  runner: 'groups',
+  reporters: ['default', '../dist/reporter.js']
+};
+```
+
+### Test with mocked traffic
+
+```shell
+$ yarn test
+```
+
+This command runs tests with mocked traffic and therefore no live traffic is enabled. You can find recordings that this command uses as mocked traffic in `/nock`.
+
+Recordings are categorised similarly as integrations in `/grid` folder:
+`nock/<scope>/<profile-name>/<provider-name>/<usecase-name>/recording-<hash>.json`
+
+- every recording gets hashed based on **input**, **test name** or **custom test name**
+- this hash can be only seen with use of `DEBUG=superface:testing:hash*`
+
+### Test with live traffic
+
+```shell
+$ yarn test:record
+```
+
+With this command, live traffic is enabled and therefore maps send requests to the live providers. This command also records this request and response. There are multiple scenarios what can happen:
+
+- use-case does not have recording → records traffic and saves as default one
+- use-case already have recording → records new traffic and compares it to the old one →
+    - if it matches → do not save recording
+    - if it doesn’t match → save new recording next to old one with suffix `-new`
+
+### Test with new mocked traffic
+
+```shell
+$ yarn test:with_new_traffic
+```
+
+This command run tests without live traffic, similar as `yarn test`, but uses new recordings instead of default ones (if present). New recordings are always newly recorded traffic that does not match the default old one and are located next to the old one with suffix `-new`.
+
+### Test and replace old recording with new one
+
+**⚠️ This command should be run only when you’re sure that tests are passing with new recordings → `yarn test:with_new_traffic`**
```

Comment:
tip: Here's a [neat trick for callouts](https://github.com/community/community/discussions/16925):

```suggestion
> **Warning**
> Run this command only when you're sure the tests pass with the new recordings (using `yarn test:with_new_traffic`).
```